src/posts/integrating-lighthouse-ci/integrate-lighthouse-ci.md

```diff
@@ -84,31 +82,30 @@ Create file at the top level of the project and add below JSON configuration.
   "ci": {
     "collect": {
       "url": [
-        "<CHANGE URL>http://localhost:8080/",
-        "<CHANGE URL>http://localhost:8080/tag/javascript",
-        "<CHANGE URL>http://localhost:8080/author/akansh",
-        "<CHANGE URL>http://localhost:8080/essential-tools-and-services-for-student-developer/"
+        "<CHANGE URL>http://localhost/",
+        "<CHANGE URL>http://localhost/tag/javascript",
+        "<CHANGE URL>http://localhost/author/akansh",
+        "<CHANGE URL>http://localhost/essential-tools-and-services-for-student-developer/"
       ],
       "numberOfRuns": 2
     },
     "assert": {
       "preset": "lighthouse:recommended",
       "assertions": {
         "first-contentful-paint": [
-          "error",
```

Comment:
don't want folks `error`'ing these unless they're very experienced and understand performance variance characteristics of their page